src/components/standalone-link/index.tsx

```diff
@@ -19,21 +19,26 @@ const StandaloneLink = ({
   textClassName,
 }: StandaloneLinkProps): ReactElement => {
   const classes = classNames(s.root, s[`color-${color}`], s[size], className)
+  const rel = openInNewTab ? 'noreferrer noopener' : undefined
+  const target = openInNewTab ? '_blank' : '_self'
 
   return (
-    <MaybeInternalLink
-      aria-label={ariaLabel}
-      className={classes}
-      download={download}
-      href={href}
-      onClick={onClick}
-      rel={openInNewTab ? 'noreferrer noopener' : undefined}
-      target={openInNewTab ? '_blank' : '_self'}
-    >
-      {iconPosition === 'leading' && icon}
-      <span className={classNames(s.text, textClassName)}>{text}</span>
-      {iconPosition === 'trailing' && icon}
-    </MaybeInternalLink>
+    <Link href={href}>
+      {/* TODO double check usage of the `onClick` prop */}
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
🗒️ Note to self: still need to double check this

Reply:
💭 Is this something that feels blocking? (If not, the `TODO` comment here seems fine by me 👍)

Reply:
Yes, it was a merge blocker for me! Just finished investigating it and updated the comment to explain why the warning was happening and why it was being disabled in this case.